resources/post-upgrade.sh

```diff
@@ -25,14 +25,16 @@ function run_postupgrade() {
   FROM_VERSION="${1}"
   TO_VERSION="${2}"
 
+  fetchDatabaseConnection
+
   echo "Executing Redmine post-upgrade from ${FROM_VERSION} to ${TO_VERSION}"
 
   if [ "${FROM_VERSION}" = "${TO_VERSION}" ]; then
     echo "FROM and TO versions are the same; Exiting..."
     exit 0
   fi
 
-  if [[ "${FROM_VERSION}" =~ ^v3.*$ || "${FROM_VERSION}" =~ ^[v]*4.((0.5-1)|(1.0-[123]))$ ]]; then
+  if versionXLessOrEqualThanY "${FROM_VERSION}" "4.1.0-3"; then
```

Comment:
This looks so nice! I can see in less than a second which versions are mentioned! Much better than a regex.

Reply:
The problem is that the `util.sh` would be the one from the source redmine, not of the newer target Redmine. Thus the functionality would not exist in older Redmines.

This is also the reason why I rewrote the SemVer functionality.